src/style.css

```diff
@@ -1 +1,46 @@
 /* add styles here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+ul,
+li {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+}
+
+.block {
+  background-color: #ffff;
+}
+
+.stars--0,
+.stars--1,
+.stars--2,
+.stars--3,
+.stars--4,
+.stars--5 {
```

Comment:
This approach is unnecessary, you should use only _stars_ instead of _stars--0, stars--1..._ and add the missing class to the blocks, as I mentioned in the previous comments